src/app/backend/handler/apihandler.go

```diff
@@ -61,6 +57,10 @@ import (
 	errorsK8s "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/client/unversioned/remotecommand"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
```

Comment:
Was that formatted by your IDE? Try change it to `goimports` style or run `goimports -w src/app/backend` in dashboard root directory. Previous format is enforced by `goimports`. I have to update `gulp format`.